spec/schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "An API to support lodging objections to compulsory and voluntary strike offs",
+    "version": "1.0.0",
+    "title": "Companies House - Strike Off Objections API"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "basePath": "/company/{companyNumber}/objections",
```

Comment:
not sure about the basePath "/company/{companyNumber}/objections"
there can be other objections, not just strike off objections
maybe "/company/{companyNumber}/objections/strike-off/request/
I admit my RESTful ness is not great, just wondering how to break it down into resources but leave it open for other objection paths in the future

Reply:
Yep. Fair enough. We have it in the repo name. Will change.